client/src/pages/CreateContest.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component, Fragment } from "react";
+import { Header } from "./Header";
+import { SignUpButton } from "./SignUpButton";
+import { CreateContestPaper } from "./CreateContestPaper";
+import { SignInPaper } from "./SignInPaper";
+import { BasicTextField } from "./TextField";
+import { SignInButton } from "./SignInButton";
+import { FormControl, makeStyles, TextField, Typography, Grid } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    "margin-top": 5,
```

Comment:
you can write snake case for more of a react-y way! `marginTop`

Reply:
will do!